README.md

```diff
@@ -6,10 +6,203 @@ developed as part of the DataWave project.
 The main working branch for this project is `datawave_ml` which was originally branched
 from the `cam6_3_139` tag.
 
+## Using this model
+
+### Obtaining CAM
+
+Clone a copy of this repository from git and ensure you checkout the `datawave_ml`
+branch on which this work is based:
+```
+git clone https://github.com/DataWaveProject/CAM.git
+cd CAM
+git checkout coupling
+```
+This branch is built upon the `cam6_3_139` tag from the
+[main ESMCOMP/CAM repository](https://github.com/ESCOMP/CAM).
+
+### Obtaining `FTorch`
+
+To use PyTorch-based neural nets in CAM, we use [`FTorch`](https://github.com/Cambridge-ICCS/FTorch) which needs to be built on the system before we build CAM.
+
+To install `FTorch` on Derecho follow the instructions in section [`FTorch` on Derecho](#ftorch-on-derecho) below.
+
+> [!NOTE]
+> The location of the `FTorch` install will be required later when building CAM.
+
+> [!NOTE]
+> If you want to build `FTorch` on another system, please follow the general instructions in the `FTorch`
+> [documentation](https://github.com/Cambridge-ICCS/FTorch).
+
+### Obtaining `FTorch`-compatible CIME
+
+This fork of CAM will use an `FTorch`-compatible version of the CIME buildsystem. This is specified under the `[cime]` section
+of the `Externals.cfg` file in the main CAM directory.
+
+To set up a CIME to use `FTorch` please follow the instructions in section [`FTorch`-compatible CIME](#ftorch-compatible-cime)
+below.
+
+### Checkout externals
+
+To fetch the external components run the following command from within the CAM root directory,
+
+```
+./manage_externals/checkout_externals
+```
+
+### Creating and running a case
+
+Details on creating a case can be found
+[here](https://ncar.github.io/CAM/doc/build/html/CAM6.0_users_guide/building-and-running-cam.html) on the NCAR website.
+For this work we are using the following testcase which can be set up by running:
+```
+./create_newcase --case <path_to_testcase_directory> --compset FMTHIST --res ne30pg3_ne30pg3_mg17 --project XXXXXXX --machine derecho
+```
+from `CAM/cime/scripts/`.
+
+You can then navigate to the case directory at `<path_to_testcase_directory>`.
+
+### Build CAM with `FTorch`
+
+Before we can run `./case.build` we first need to make some manual changes to the `Makefile` located in
+`<path_to_testcase_directory>/Tools/Makefile`. This will allow the CIME build system to locate `FTorch`.
+
+From the test case directory, modify `Tools/Makefile` line 602 to set the environment variable `FTORCH_LIB` to the location of
+the `FTorch` library on your system.
+
+If you followed the instructions in section [`FTorch` on Derecho](#ftorch-on-derecho) this will be `$HOME/FTorch/bin/ftorch_intel`.
+
+```make
+FTORCH_LIB := $HOME/FTorch/bin/ftorch_intel
```

Comment:
Do we want to alter this to the full glade path instead of $HOME as was required to get my build working properly?

Reply:
Does https://github.com/DataWaveProject/CAM/pull/5/commits/5cfb8f925efb50eaa91dd907df7436a47ab2f74d resolve this?